spring-integration-core/src/main/java/org/springframework/integration/endpoint/ExpressionEvaluatingMessageSource.java

```diff
@@ -47,4 +47,8 @@ public T doReceive() {
 		return this.evaluateExpression(this.expression, this.expectedType);
 	}
 
+	public Expression getExpression() {
```

Comment:
`@since 4.3` ?
And short JavaDocs to make happy CheckStyle :smile: 

Will fix on merge if only concern.